src/app/util/attribute-table.cpp

```diff
@@ -280,7 +322,10 @@ EmberAfStatus emAfWriteAttribute(EndpointId endpoint, ClusterId cluster, Attribu
         // The callee will weed out attributes that do not need to be stored.
         emAfSaveAttributeToStorageIfNeeded(data, endpoint, cluster, metadata);
 
-        MatterReportingAttributeChangeCallback(endpoint, cluster, attributeID);
+        if (markDirty != MarkAttributeDirty::No)
```

Comment:
Where is the write path for AttributeAccessInterface-backed attributes? Will it ever have the ::No ?

Reply:
I'm not sure how to handle AttributeAccessInterface yet, because for those we don't really have a sane way of "get the current value"...  Needs some thought.

The No option here is basically to lay the groundwork for Q quality support.  For AttributeAccessInterface that would correspond to "change your member and do not call the mark-dirty function"...